### PR TITLE
Add project sorting to front end

### PR DESCRIPTION
**Pull request by charliecrouse**

### CODE REVIEW

**Comment by charliecrouse** [comment]
Technically this is still sorting in the backend (just during the rendering phase). But all of the sorting logic has abstracted away from the controllers.

File: lamby/models/project.py [hunk 1]
```diff
@@ -28,6 +30,14 @@ class Project(db.Model):
     # README -- Project information stored in markdown format
     readme = db.Column(db.Text, default='# README')
 
+    # DATE CREATED -- Timestamp of when the project was created
```
Comment (pkoushik) [inline lamby/models/project.py]: Also wow it's that easy.

File: lamby/static/js/iterator.js [hunk 2]
```diff
@@ -1,17 +1,20 @@
+/* eslint-env browser */
```
Comment (pkoushik) [inline lamby/static/js/iterator.js]: i c u

File: lamby/database/stub.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+"""
```
Comment (pkoushik) [inline lamby/database/stub.py]: Ok, this is pretty cool.

File: lamby/models/project.py (hunk 1 above)
Comment (charliecrouse) [reply lamby/models/project.py]: yes it took a little digging tho

File: lamby/database/stub.py (hunk 3 above)
Comment (charliecrouse) [reply lamby/database/stub.py]: We can just keep it internal for now because it doesn't add any onnx files.  Just dummy data that i used to test sorting

Comment (pkoushik) [reply lamby/database/stub.py]: Ok cool.

**Pull request merged**